src/modules/configurator/components/saved-configurations/Configurations.tsx

```diff
@@ -0,0 +1,19 @@
+import ConfigureLinkButton from "../ConfigureLinkButton";
+import ConfigurationsEmpty from "./ConfigurationsEmpty";
+import { useRecoilState } from "recoil";
+import { userConfigurationsState } from "../../state/configurationsState";
+import ConfigurationsSaved from "./ConfigurationsSaved";
+
+export default function Configurations() {
+  const [userConfigurations] = useRecoilState(userConfigurationsState);
+
+  return (
+    <section className="text-center md:text-left px-10 sm:px-20 md:px-20 lg:px-36 py-16 flex flex-col">
+      <header className="flex flex-col sm:flex-row justify-between">
+        <h2 className="mb-6 text-lg sm:text-sm">View saved configurations</h2>
+        <ConfigureLinkButton path="/configure-a-car" title="Configure a car" />
+      </header>
+      {!userConfigurations ? <ConfigurationsEmpty /> : <ConfigurationsSaved />}
```

Comment:
userConfigurations can be undefined or empty array? Check than if the array is also empty and display the empty state.

Reply:
Set default value for userConfigurationsState to empty array, removed check for if userConfigurations are undefined and added check for if userConfigurations array is empty.